firebase-firestore/src/androidTest/java/com/google/firebase/firestore/ValidationTest.java

```diff
@@ -306,7 +306,7 @@ public void transactionsRequireCorrectDocumentReferences() {
                             try {
                               transaction.get(badRef);
                             } catch (FirebaseFirestoreException e) {
-                              fail("transaction.get() triggered wrong exception: " + e);
+                              fail("transaction.get() triggered wrong exception: %s", e);
```

Comment:
Shouldn't this also be fail(e, "transaction.get ...")?

Reply:
Oops; there's two mistakes here:

> Shouldn't this also be fail(e, "transaction.get ...")?
Mistake 1: yeah. Or it would be except:

2: This is actually junit's fail() method, not Assert.fail(), and thus, this line should've been unchanged. Fixed.

Reply:
I think there's a few more of these too. I'll recheck *Test.java.

Reply:
Done.